src/lerner_lab_to_nwb/seiler_2024/seiler_2024fiberphotometryinterface.py

```diff
@@ -81,171 +83,286 @@ def add_to_nwbfile(
+            manufacturer="Doric Lenses",
+            model="Fiber Optic Implant",
+            numerical_aperture=0.48,
+            core_diameter_in_um=400.0,
+        )
+
+        # Excitation Sources
+        dms_signal_excitation_source = ExcitationSource(
+            name="dms_signal_excitation_source",
+            description="465nm and 405nm LEDs were modulated at 211 Hz and 330 Hz, respectively, for DMS probes. 465nm and 405nm LEDs were modulated at 450 Hz and 270 Hz, respectively for DLS probes. LED currents were adjusted in order to return a voltage between 150-200mV for each signal, were offset by 5 mA, were demodulated using a 4 Hz lowpass frequency filter.",
+            manufacturer="Doric Lenses",
+            model="Connectorized LED",
+            illumination_type="LED",
+            excitation_wavelength_in_nm=465.0,
+        )
+        dms_reference_excitation_source = ExcitationSource(
+            name="dms_reference_excitation_source",
             description="465nm and 405nm LEDs were modulated at 211 Hz and 330 Hz, respectively, for DMS probes. 465nm and 405nm LEDs were modulated at 450 Hz and 270 Hz, respectively for DLS probes. LED currents were adjusted in order to return a voltage between 150-200mV for each signal, were offset by 5 mA, were demodulated using a 4 Hz lowpass frequency filter.",
+            manufacturer="Doric Lenses",
+            model="Connectorized LED",
+            illumination_type="LED",
+            excitation_wavelength_in_nm=405.0,
         )
+        dls_signal_excitation_source = ExcitationSource(
+            name="dls_signal_excitation_source",
+            description="465nm and 405nm LEDs were modulated at 211 Hz and 330 Hz, respectively, for DMS probes. 465nm and 405nm LEDs were modulated at 450 Hz and 270 Hz, respectively for DLS probes. LED currents were adjusted in order to return a voltage between 150-200mV for each signal, were offset by 5 mA, were demodulated using a 4 Hz lowpass frequency filter.",
+            manufacturer="Doric Lenses",
+            model="Connectorized LED",
+            illumination_type="LED",
+            excitation_wavelength_in_nm=465.0,
+        )
+        dls_reference_excitation_source = ExcitationSource(
+            name="dls_reference_excitation_source",
+            description="465nm and 405nm LEDs were modulated at 211 Hz and 330 Hz, respectively, for DMS probes. 465nm and 405nm LEDs were modulated at 450 Hz and 270 Hz, respectively for DLS probes. LED currents were adjusted in order to return a voltage between 150-200mV for each signal, were offset by 5 mA, were demodulated using a 4 Hz lowpass frequency filter.",
+            manufacturer="Doric Lenses",
+            model="Connectorized LED",
+            illumination_type="LED",
+            excitation_wavelength_in_nm=405.0,
+        )
+
```

Comment:
Again here you don't need to define dms and dls excitation_source because they are the same source just commanded in a different way, that you can describe by creating two different commanded voltages series.